Content.Client/_DV/CartridgeLoader/Cartridges/NanoChatLookupUiFragment.xaml.cs

```diff
@@ -0,0 +1,66 @@
+using System.Text;
+using Content.Shared._DV.CartridgeLoader.Cartridges;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client._DV.CartridgeLoader.Cartridges;
+
+[GenerateTypedNameReferences]
+public sealed partial class NanoChatLookupUiFragment : BoxContainer
+{
+    public NanoChatLookupUiFragment()
+    {
+        IoCManager.InjectDependencies(this);
```

Comment:
there are no dependencies, this is copy pasted

Reply:
this is copy pasted